packages/@ourworldindata/utils/src/MarkdownTextWrap/MarkdownTextWrap.tsx

```diff
@@ -621,6 +621,14 @@ export class MarkdownTextWrap extends React.Component<MarkdownTextWrapProps> {
         return splitIntoLines(tokensWithReferenceNumbers, this.maxWidth)
     }
 
+    @computed get width(): number {
+        const { htmlLines } = this
+        const lineLengths = htmlLines.map((tokens) =>
+            tokens.reduce((acc, token) => acc + token.width, 0)
```

Comment:
```suggestion
            sumBy(tokens, (token) => token.width)
```

This is potentially clearer to read, but up to you really.